src/core/middleware.py

```diff
@@ -117,6 +118,15 @@ def process_request(request):
             request.site_type = request.site_object = press
             request.model_content_type = ContentType.objects.get_for_model(press)
             request.press_base_url = press.site_url()
+
```

Comment:
If this doesn't change very often it might be worth caching `get_search_data_file` otherwise it will be hit on every request.

Reply:
@joemull  With If we turn the function `get_search_data_file` into `get_search_data_url` and return instead `docs_file.url` we would only have to cache that string